web/src/reducers/dataReducer.test.js

```diff
@@ -98,15 +98,15 @@ test('Grid is expired', () => {
 test('Exchange data is fetched', () => {
   let state = undefined;
   state = reducer.getInitialState();
-  expect(state.exchanges['DK-DK2->SE'].config).toHaveProperty('sortedCountryCodes');
-  expect(state.exchanges['DK-DK2->SE'].data).toHaveLength(0);
+  expect(state.exchanges['DE->DK-DK1'].config).toHaveProperty('sortedCountryCodes');
+  expect(state.exchanges['DE->DK-DK1'].data).toHaveLength(0);
```

Comment:
Cause the state files used for the mockserver don't have any SE-SE* data yet so it will always fail and if I'm not mistaken we need to merge this to get that data...